### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by luvjOi**

### CODE REVIEW

File: src/pages/calendar-dropdown/calendar-dropdown.ts [hunk 1]
```diff
@@ -29,7 +29,6 @@ export class CalendarDropdownPage {
 @Component({
```
Comment (clblamires) [inline src/pages/calendar-dropdown/calendar-dropdown.ts]: Remove this entire component. It doesn't do anything. (see the button text "Learn Ionic" and "Documentation", etc.). Those are incorrect for this app.

**Comment by luvjOi** [comment]
Oh you can delete the calendar-dropdown files in the actual page folder, I was trying to update the whole calendar-dropdown folder but the site just dumped them into the pages folder and I don’t know how to delete them off of it. Thanks.

-Lauren

File: src/pages/calendar-dropdown/calendar-dropdown.ts (hunk 1 above)
Comment (luvjOi) [reply src/pages/calendar-dropdown/calendar-dropdown.ts]: I guess I’m confused, I already removed the @Component -class myPage stuff. I’m probably looking at the wrong thing but if I remove anything rise from the page, I’ll get an error. Do you mean the import from ionic-angular stuff? That’s what the node.js command window is saying is unused.

Comment (luvjOi) [reply src/pages/calendar-dropdown/calendar-dropdown.ts]: Did you mean this code in the calendar.ts file? I removed that as a test on my local version and things were still fine.

@Component({
	 template: `
    <ion-list>
      <button ion-item (click)="close()">Learn Ionic</button>
      <button ion-item (click)="close()">Documentation</button>
      <button ion-item (click)="close()">Showcase</button>
      <button ion-item (click)="close()">GitHub Repo</button>
    </ion-list>
   `
})

**Pull request closed without merging**